src/bot.py

```diff
@@ -12,7 +11,6 @@ def markdown(text):
 
 
 @tb.message_handler(commands=['help', 'start', 'hi', 'hello'])
-@tb.message_handler(func=lambda m: m.text in ['Hi', 'H'])
```

Comment:
not sure it is okay to remove this on. People usually type just help but not /help.
So, I expand it here #3